paper/paper.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'ODE Test Problems'
+output:
+  rmarkdown::html_vignette:
+    keep_md: TRUE
+tags:
+- Octave
+- Matlab
+- Initial Value Problems
+- Ordinary Differential Equations
+- Differential algebraic equations
+
+authors:
+  - name: Steven Roberts
+    orcid: 0000-0002-7225-2501
+    corresponding: true # (This is how to denote the corresponding author)
+    equal-contrib: true
+    affiliation: "1" # (Multiple affiliations must be quoted)
+  - name: Andrey A. Popov
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    orcid: 0000-0002-7726-6224
+    affiliation: 2
+  - name: Arash Sarshar
+    affiliation: 3
+    orcid: 0000-0002-6633-8915
+  - name: Adrian Sandu
+    affiliation: 3
```

Comment:
Put orc-id

Reply:
it exists, but  is not populated, what do you think ? 
[0000-0002-5380-0103](https://orcid.org/0000-0002-5380-0103)

Reply:
Yes, I was informed to use that.